index.html

```diff
@@ -8,6 +8,96 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+   <div id="home-page"> 
+        <div id="top-info">
+            <div id="top-left-info">
+                <div class="left-info">Travel</div>
+                <div class="left-info">Goo</div>
+            </div>
+            <div id="top-mid-info">
+               <div class="mid-info">Home</div>
```

Comment:
Home, about us etc. should be using <a> element.

Reply:
Technically, an element is a collection of start tag, attributes, end tag, content between them.

https://www.javatpoint.com/html-elements

pardon ... i cant get you ...

Reply:
They should be using 'a' element